suite-common/suite-config/src/fiat.ts

```diff
@@ -64,3 +64,106 @@ export const FIAT = {
         'xau',
     ],
 } as const;
+
+export type CurrencyType =
+    | 'usd'
+    | 'eur'
+    | 'gbp'
+    | 'aed'
+    | 'ars'
+    | 'aud'
+    | 'bdt'
+    | 'bhd'
+    | 'bmd'
+    | 'brl'
+    | 'cad'
+    | 'chf'
+    | 'clp'
+    | 'cny'
+    | 'czk'
+    | 'dkk'
+    | 'hkd'
+    | 'huf'
+    | 'idr'
+    | 'ils'
+    | 'inr'
+    | 'jpy'
+    | 'krw'
+    | 'kwd'
+    | 'lkr'
+    | 'mmk'
+    | 'mxn'
+    | 'myr'
+    | 'nok'
+    | 'nzd'
+    | 'php'
+    | 'pkr'
+    | 'pln'
+    | 'rub'
+    | 'sar'
+    | 'sek'
+    | 'sgd'
+    | 'thb'
+    | 'try'
+    | 'twd'
+    | 'vef'
+    | 'vnd'
+    | 'zar'
+    | 'xdr'
+    | 'xag'
+    | 'xau';
+
+export type Currency = {
+    label: CurrencyType;
+    value: string;
+};
+export type CurrencyMap = Record<CurrencyType, Currency>;
+
+export const fiatCurrenciesMap: CurrencyMap = {
```

Comment:
`Map` suffix is misleading IMHO, it doesn't map anything it's simple definition so I think name `fiatCurrencies` is better. 

I would also prefer to refactor this whole file to avoid duplicities. Currenies shortcuts it's defined like 3x here in this file.

My suggestion is to use just this single object as source truth and only place where currencies are defined.

```tsx
export const fiatCurrencies = {
     usd: { label: 'usd', value: 'United States Dollar' },
     eur: { label: 'eur', value: 'Euro' },
     ....
} as const;

export type FiatCurrency = typeof keyof fiatCurrencies;

export const FIAT = {
    tickers: [ ... ],
    currencies: Object.keys(fiatCurrencies);
} as const 
```

Reply:
I agree with using Object.keys, for the FIAT but I'm not too sure about why removing the type for keys? Having it strongly typed and defined as only these options are valid, doesn't it give us stricter way of updating the map object? If I define the type for keys, then if someone makes a typo in the label, it doesn't correspond with the type anymore and makes it visible that this is not wanted. With type assertion, this additional check is not possible anymore. I know it looks like duplicity, but gives us additional check and kind of static test that this object is in the correct format.

Reply:
If I do the `typeof keyof ...`, I will have to use string as type of label property which imo is too generic for something like this. I would prefer there to be type with only allowed properties to have stricter type checking and avoid unnecessary bugs.

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/088ccfb38beef6c123b6dd7a50d6ae4c2f120a07

Reply:
You need to do `as const` for `fiatCurrencies`, then `typeof keyof` will get specific keys and not only `string`

About the typo in the label property for example. If you define it `as const` you can try what will happen if you do some typo here, I am pretty sure it will throw error TS error in places where it is actually used, so it's strongly typed. Maybe not ideal place to throw error, but it's super simple to define it and that's reasonable trade-of.

Doing:
```tsx
type Currency = "usd" | "czk"
const fiatCurrencies: Record<Currency, whaever> = {
   usd: ...,
   czk: ...,
}
```
Have no sense in term of strong typing. You can do type while defining `type Currency = "uds" | "kcz"` and autocomplete will autocomplete that later and you can miss it anyway, so it's better to write it only once that duplicate it.